### ISSUE TITLE
Slow render of catalog pics TTFB (in SPA web app)

### ISSUE THREAD

**Issue by Kaidanov** [dotnet-architecture/eShopOnContainers#520]
Rendering of the bundle is very slow , although through the just url  I am getting quicker results. 
(still slow but better)
How can I improve it?

4-7 sec with bundles
![image](https://user-images.githubusercontent.com/164266/36531333-2876849c-17c6-11e8-8ca7-619f39d5608d.png)

2.5 sec without - just calling to url
![image](https://user-images.githubusercontent.com/164266/36531436-8393b08e-17c6-11e8-931c-c69aea67d0cc.png)

And those are small images.

**Comment by raboud** [comment]
Tbis doesn't totally address your concerns but they should probably be marked cached.

**Comment by mvelosop** [comment]
Hi @Kaidanov 

This is a bit old, but just did a quick test and found a **BIG** difference between running the app inside VS with [F5] and running it from the CLI (always resetting the browser cache).

I tried with the MVC app because I could not get the SPA running inside VS right now, but the time for MVC and SPA with the CLI are very similar:

VS with [F5] (MVC): **25.30s**
![image](https://user-images.githubusercontent.com/7357806/40501860-08f50492-5f81-11e8-960d-15bdafdc5916.png)

These times are even a bit worse that what you got.

CLI (MVC): **1.72s**
![image](https://user-images.githubusercontent.com/7357806/40502356-a06f76f8-5f82-11e8-85b1-7e09cbf52ddc.png)

CLI (SPA): **1.98s**
![image](https://user-images.githubusercontent.com/7357806/40502205-2b8279a8-5f82-11e8-8aec-dabd257806b5.png)

The first time I tried on the CLI it took a little over 4 seconds, from the second time on I got consistently the times shown, I guess there's the effect of some caching at the DB and/or disk level.

This just seems to be the effect of the necessary overhead for debugging.
